### DISCUSSION TITLE
[Feature] Only show server images on app

### DISCUSSION THREAD

**Discussion by pokemonbattledroid** [immich-app/immich#4656] [Feature Request]
### Enable an option to select folders to display in the app - ie omit showing local media and only show server content

**Firstly, immich - awesome.  It's beyond amazing.**

Is there a way to use the mobile app to **only** view media on the immich server?  I'd rather not see random whatsapp, signal, messenger gifs and photos.  If there was a way to turn on/off folders to view on the mobile would be useful.  Essentially just viewing server content.

Hope this makes sense.  Of course I can just view the web UI from the mobile, but it isn't as nice as the app!!!!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by aviv926** [comment]
Yes you can, in the immich app you have to choose folders to back up an empty folder or alternatively create a folder on the phone that only has one photo
After that you will see in the timeline only the images that are on your server and another 1 image from the folder you selected
hope it helps

**Reply by pokemonbattledroid** [reply]
Hi,
Thanks for the reply.  Oh, I didn't realise this.  So it's sort of a hack I suppose?  What you are saying is:
- create empty folder
- select empty folder as backup folder
- if no photos, then immich app will just show images on the server
I'll give that a try.  It would be great though if you can select folders or just choose as an option in the future perhaps.

Thanks!

**Reply by aviv926** [reply]
> Hi,
> Thanks for the reply.  Oh, I didn't realise this.  So it's sort of a hack I suppose?  What you are saying is:
> - create empty folder
> - select empty folder as backup folder
> - if no photos, then immich app will just show images on the server
> I'll give that a try.  It would be great though if you can select folders or just choose as an option in the future perhaps.
> 
> Thanks!

Yes, it's totally like that :)

I had a similar question in Discord with the owner of the project, this is what he suggested I do
